ROMS.md

```diff
@@ -0,0 +1,188 @@
+# ROMS 3.6 Porting Guide(CentOS 7.6)
+## Introduction
+Regional Ocean Modeling System is a three-dimensional regional ocean model, jointly developed by the Institute of Marine and Coastal Sciences of Rutgers University and the University of California, Los Angeles. It is widely used to simulate the hydrodynamic and water environment of the ocean and estuarine region.
+
+For more information about the ROMS, visit the official ROMS website.
+
+Programming language: Fortran
+
+Brief description: three-dimensional regional ocean model
+
+Open-source protocol: MIT/X License
+
+##### Recommended Version
+The recommended version is ROMS V3.6
+## Environment Requirements
+### Software Requirements
+| Item         | Version  |  Download Address                                                          |
+| ------------ | -------- | -------------------------------------------------------------------------- |
+|  HDF5        |  1.10.6  |  https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.6/src/ |
+|  PNETCDF     |  1.12.1  |  https://parallel-netcdf.github.io/wiki/Download.html                      |
+|  NETCDF-C    |  4.7.3   |  https://github.com/Unidata/netcdf-c/releases/tag/v4.7.3                   |
+|  NETCDF-F    |  4.5.2   |  https://github.com/Unidata/netcdf-fortran/releases/tag/v4.5.2             |
+|  ROMS        |  3.6     |  https://www.myroms.org/svn/src/trunk                                      |
+### OS Requirements
+| Item    | Version     | How to Obtain                     |
+| ------- | ----------- | --------------------------------- |
+| CentOS  | 7.6         |  https://www.centos.org/download/ |
+| Kernel  | 4.14.0-115  |  Included in the OS image.        |
+## Configuring the Compilation Environment
+### Installing GNU 9.3
+
+
+    yum install -y centos-release-scl
+    yum install -y devtoolset-9-gcc
+    yum install -y devtoolset-9-gcc-c++
+    yum install -y devtoolset-9-binutils
+    scl enable devtoolset-9 bash
+    echo "souce /opt/rh/devtoolset-9/enable" >> /etc/profile
+### Installing Open MPI
```

Comment:
Add Open MPI to the table with dependencies

Reply:
done